latest/index.bs

```diff
@@ -218,6 +219,8 @@ to the current zarr group. The "path"s MUST be ordered from largest (i.e. highes
 It MUST contain the field "axes", which is a list of dimension names of the axes.
 The values MUST be unique and one of `{"t", "c", "z", "y", "x"}`.
 The number of values MUST be the same as the number of dimensions of the arrays corresponding to this image.
+In addition, the "axes" values MUST be repeated in the field "_ARRAY_DIMENSIONS" of the top-level group for the arrays containing
+the multiscale data. This ensures compatibility with the [xarray zarr encoding](http://xarray.pydata.org/en/stable/internals/zarr-encoding-spec.html#zarr-encoding).
```

Comment:
@thewtex If I understand the xarray encoding correctly this addition should ensure that we are compatible.

Reply:
@constantinpape yes, you are right!

One tweak: *top-level group* -> *all scale groups*. This ensures xarray compatibility and that each scale can be treated independently.